doc/API-Entities.md

```diff
@@ -1001,6 +1008,14 @@ Mutually exclusive with <code>data</code> <code>datasize</code>.
 </td>
 </tr>
 
+<tr>
+<td><code>scales</code></td>
+<td>Array of Photo Scales</td>
+<td>
+List of Scale objects listing the id, scale, link, etc. of each scale
```

Comment:
Is it for both attached and embedded pictures? And no need to start listing the attributes here as the Photo Scale entity is described later.

Reply:
I don't understand the question.

Reply:
In Friendica pictures can either be attached to posts, they display the same way they do in Mastodon in a carousel at the bottom of the post, or be embedded (or inline) in the post body itself. I'm guessing this attribute is only about the attached pictures, not about the embedded ones.

Reply:
Ah, sorry, I thought this was in the Status structure.